### ISSUE TITLE
Build fails for .NET MAUI class library with .NET MAUI Essentials targeting the Windows platform

### ISSUE THREAD

**Issue by egvijayanand** [egvijayanand/dotnet-maui-templates#109]
Build fails for .NET MAUI class library with .NET MAUI Essentials targeting the Windows platform.

**Template Package Version:**

3.1.0-preview.2

**Error message:**

D:\NuGet\packages\Microsoft.WindowsAppSDK\1.2.230118.102\buildTransitive\MrtCore.PriGen.targets(380,5): error MSB4062: The "Microsoft.Build.AppxPackage.GetSdkFileFullPath" task could not be loaded from the assembly C:\Program Files\dotnet\sdk\8.0.100-preview.1.23115.2\Microsoft\VisualStudio\v17.0\AppxPackage\Microsoft.Build.AppxPackage.dll. Could not load file or assembly 'C:\Program Files\dotnet\sdk\8.0.100-preview.1.23115.2\Microsoft\VisualStudio\v17.0\AppxPackage\Microsoft.Build.AppxPackage.dll'. The system cannot find the path specified. Confirm that the <UsingTask> declaration is correct, that the assembly and all its dependencies are available,and that the task contains a public class that implements Microsoft.Build.Framework.ITask. [E:\SampleLib\SampleLib.csproj::TargetFramework=net8.0-windows10.0.19041.0]

**Comment by egvijayanand** [comment]
When making use of .NET MAUI Essentials and targeting the Windows platform, it is necessary to define the `EnableMsixTooling` project option to resolve the build issue.

Will get fixed in the upcoming preview (3.1.0-preview.3).

```xml
<Project Sdk="Microsoft.NET.Sdk">
  <PropertyGroup>
    <UseMauiEssentials>true</UseMauiEssentials>
    <EnableMsixTooling>true</EnableMsixTooling>
  </PropertyGroup>
</Project>
```

**Comment by egvijayanand** [comment]
Fix released via [v3.1.0-preview.3](https://www.nuget.org/packages/VijayAnand.MauiTemplates/3.1.0-preview.3).
